src/Online_Bookstore/Dashboard/BookDetail.js

```diff
@@ -0,0 +1,28 @@
+import React, { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { useNavigate } from "react-router-dom";
```

Comment:
Why import 2 hooks from the same package twice?

Simply, `import { useParams, useNavigate } from "react-router-dom";`

@Poorvi-3pillar